django-web-app/cogniguard/settings.py

```diff
@@ -24,11 +24,13 @@
 # PJ_SECRET_KEY=os.getenv('PJ_SECRET_KEY','')
 
 # for vercel deployment
-SECRET_KEY = os.environ.get('PJ_SECRET_KEY')
+#SECRET_KEY = os.environ.get('PJ_SECRET_KEY')
+SECRET_KEY = "default-development-secret-key"
```

Comment:
You have to undo the development server changes, read the readme carefully, 
this is causing deployment failure

change debug to false and set the environ secret key